src/terminal/input/terminalInput.hpp

```diff
@@ -69,14 +70,28 @@ namespace Microsoft::Console::VirtualTerminal
         std::optional<wchar_t> _leadingSurrogate;
 
         std::optional<WORD> _lastVirtualKeyCode;
+        DWORD _lastControlKeyState = 0;
+        uint64_t _lastLeftCtrlTime = 0;
+        uint64_t _lastRightAltTime = 0;
+        std::unordered_map<int, std::wstring> _keyMap;

```

Comment:
`til::linear_flat_set` would probably work quite nicely for such a map. But realistically I doubt anyone would notice that performance difference. 😅

Reply:
If it was a drop-in replacement I'd be fine with that, but it seems like it would be a bit of a pain to incorporate, so unless you're super keen on the idea, I'd be inclined to leave it as it is.

I also considered maybe using an array for this, if performance was an issue, but that seemed like it might be a waste of memory.